app/controllers/api/v1/soap_api_controller.rb

```diff
@@ -7,7 +7,7 @@ class Api::V1::SoapApiController < Api::V1::BaseController
 
   before_action :load_adapter
 
-  soap_action 'get_final_cites_certificate',
+  soap_action :get_final_cites_certificate,
```

Comment:
For some reasons this works in development, but when I deployed to staging the server would not even start, giving some error from within wash out to do with undefined operation for symbol; I checked that when method names are strings it works. If you want to change it back to symbols can you make sure it works in staging? not fully sure what the problem there was, but the docs for wash_out have examples with strings so I thought it's safer to just follow that.